Panic at some point

Hi @kpetremann,
Thank you for creating salt-exporter, really great work!

If you have time, could you please take a look at this panic message?

```
panic: interface conversion: interface {} is []interface {}, not string

goroutine 12 [running]:
github.com/kpetremann/salt-exporter/pkg/events.extractStateFromArgs(...)
	/home/runner/work/salt-exporter/salt-exporter/pkg/events/events.go:47
github.com/kpetremann/salt-exporter/pkg/events.(*SaltEvent).ExtractState(0xc0000a2b60?)
	/home/runner/work/salt-exporter/salt-exporter/pkg/events/events.go:61 +0x47d
github.com/kpetremann/salt-exporter/internal/metrics.ExposeMetrics({0x980978, 0xc000106dc0}, 0xc000072240)
	/home/runner/work/salt-exporter/salt-exporter/internal/metrics/metrics.go:66 +0x89d
created by main.main
	/home/runner/work/salt-exporter/salt-exporter/main.go:62 +0x53e
```

Probably, events with incorrect schema should be just ignored.

--- a/pkg/events/events.go
+++ b/pkg/events/events.go
@@ -41,10 +41,14 @@ func extractStateFromArgs(args interface{}, key string) string {
 	if v, ok := args.(string); ok {
 		return v
 	}
+
 	// kwargs
 	if v, ok := args.(map[string]interface{}); ok {
-		if _, ok := v[key]; ok {
-			return v[key].(string)
+		if _, keyExists := v[key]; !keyExists {
+			return ""
+		}
+		if ret, isString := v[key].(string); isString {
+			return ret
 		}
 	}
 

